SimpleAPI.rst:48: [broken] https://app-eu.earthsquad.global: 500 Server Error: Internal Server Error for url: https://app-eu.earthsquad.global/api/admin/login/?next=/api/admin/
SimpleAPI.rst:48: [broken] https://app-eu.earthsquad.global/api/rest-auth/login: 500 Server Error: Internal Server Error for url: https://app-eu.earthsquad.global/api/rest-auth/login/
articles/SeleniumGridV4WithBaangt.rst:4: [broken] https://baangt.org: HTTPSConnectionPool(host='baangt.org', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError("hostname 'baangt.org' doesn't match 'www.baangt.org'")))
changelog.rst:64: [broken] https://baangt.jointhedocs.io: HTTPSConnectionPool(host='baangt.jointhedocs.io', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f8ac880da50>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known'))
docs/baangt.TestSteps.rst:1: [broken] http://$(BASEURL)/$(ENDPOINT: HTTPConnectionPool(host='$(baseurl)', port=80): Max retries exceeded with url: /$(ENDPOINT (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8ac87a7fd0>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known'))
